api-doc/openapi.yaml

```
@@ -1,48 +1,514 @@
+  /donation/{donationsId}:
+    parameters:
+      - name: donationsId
+        in: path
+        required: true
+        description: Donation id (uuid).
+        schema:
+          type: integer
+          format: int64
+          minimum: 1
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/DonationsOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Donations'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+  /gamedata:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/GameDataAllOk'
+  /gamedata/{gamedataId}:
+    parameters:
+      - name: gamedataId
+        in: path
+        required: true
+        description: Game data id (uuid).
+        schema:
+          type: integer
+          format: int64
+          minimum: 1
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/GameDataOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GameData'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+  /stream_metadata:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/StreamMetadataOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StreamMetadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+      security:
+        - ApiKeyAuth: []
+  /player_info:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/PlayerInfoOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PlayerInfo'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+      security:
+        - ApiKeyAuth: []
+  /incentives:
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/IncentivesAllOk'
+  /incentive/{incentiveId}:
```

I think this is a cleaner api design. Same with other endpoints
```suggestion
  /incentives/{incentiveId}:
```

---

I agree, updated endpoint names